Nitrox.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
Is it possible to leave this file untouched?

Reply:
Yes, I just figured we would want the projects to be part of the solution for easy management. If not I can revert this file.